webapp/app/containers/Widget/components/Chart/Table/index.tsx

```diff
@@ -113,17 +116,37 @@ export class Table extends React.PureComponent<IChartProps, ITableStates> {
     onShowSizeChange: this.onPaginationChange
   }
 
-  public componentDidMount () {
+  public componentDidMount() {
     const { headerFixed, withPaging } = this.props.chartStyles.table
     this.adjustTableCell(headerFixed, withPaging)
   }
 
-  public componentDidUpdate () {
-    const { headerFixed, withPaging } = this.props.chartStyles.table
+  public componentDidUpdate() {
+    const { data, chartStyles, width } = this.props
+    const { headerFixed, bordered, withPaging, size, autoPlay, palyInterval } = chartStyles.table
     this.adjustTableCell(headerFixed, withPaging, this.state.tablePagination.total)
+
+    const { tablePagination, tableColumns, tableBodyHeight, mapTableHeaderConfig } = this.state
```

Comment:
When comes to any props changed that to this `Table` component, `componentDidUpdate` will always be executed, and in this case, the `timer` will also be initialized again, so the interval that refreshes the pagination will not be correct.

Reply:
Every time the last `timer` is cleared, only the unique `timer` instance exists.